metrics/sinks/stackdriver/stackdriver.go

```diff
@@ -163,10 +104,20 @@ func (sink *StackdriverSink) Stop() {
 func (sink *StackdriverSink) processMetrics(metricValues map[string]core.MetricValue,
 	timestamp time.Time, labels map[string]string, createTime time.Time) []*sd_api.TimeSeries {
 	timeseries := make([]*sd_api.TimeSeries, 0)
-	for name, value := range metricValues {
-		ts := sink.TranslateMetric(timestamp, labels, name, value, createTime)
-		if ts != nil {
-			timeseries = append(timeseries, ts)
+	if useOldResourceModel {
+		for name, value := range metricValues {
+			ts := sink.TranslateMetric(timestamp, labels, name, value, createTime)
+			if ts != nil {
+				timeseries = append(timeseries, ts)
+			}
+		}
+	}
+	if useNewResourceModel {
+		for name, value := range metricValues {
+			ts := sink.TranslateNewMetric(timestamp, labels, name, value, createTime)
```

Comment:
If you use the `legacy` notation when listing metrics, use the same notation here

Old method should be named `LegacyTransalteMetric`, new one -- juse `TranslateMetric`

Reply:
Done